src/sentry/api/base.py

```diff
@@ -192,6 +193,8 @@ def _create_api_access_log(self):
                 path=self.request.path,
```

Comment:
this function is also becoming quite big, wonder if its worth moving to a file in  `src/sentry/utils` or to`src/sentry/api/utils.py`, and potentially adding some basic tests somewhere.